src/main/scala/mesosphere/marathon/api/v2/PodsResource.scala

```diff
@@ -93,7 +93,7 @@ class PodsResource @Inject() (
       implicit val identity = await(authenticatedAsync(req))
       val podRaml = unmarshal(body)
 
-      val roleSettings = RoleSettings.forService(config, PathId(podRaml.id), groupManager.rootGroup())
+      val roleSettings = RoleSettings.forService(config, PathId(podRaml.id).canonicalPath(PathId.root), groupManager.rootGroup())
```

Comment:
One more instance of the bug

Reply:
Is this just the absolute path of `podRaml.id`? So rather `PathId(podRaml.id).asAbosultePath`?